src/main/java/org/elasticsearch/percolator/PercolateContext.java

```diff
@@ -693,4 +690,79 @@ public InnerHitsContext innerHits() {
         throw new UnsupportedOperationException();
     }
 
+    // TODO needs real implementation?
```

Comment:
did this when putting something in context or the headers or when calling the `copy*()` methods... doesnt really matter for reading methods IMO